src/gtclang/Driver/Options.inc

```diff
@@ -46,7 +46,8 @@ OPT(std::string, ConfigFile, "", "config", "",
 OPT(std::string, Backend, "gridtools", "backend", "", 
     "Set backend code-generator. Possible values for <backend> are:"
     "\n - gridtools = gridtools C++ code"
-    "\n - c++       = naive C++ code\n", "<backend>", true, false)
+    "\n - c++-naive   = naive C++ code\n"
+    "\n - c++-opt     = optimized C++ code\n", "<backend>", true, false)
```

Comment:
the last \n should not be here, creates extra empty line

Reply:
done